tests/php/Forms/HTMLEditor/HTMLEditorSanitiserTest.php

```diff
@@ -59,5 +83,8 @@ public function testSanitisation()
 
             $this->assertEquals($output, $htmlValue->getContent(), $desc);
         }
+
+        // reset configurable value
+        $sanitiser->config('link_noopener_value')->set('link_noopener_value', $defaultValue);
```

Comment:
This is actually unnecessary (as is the fetching and storing of the original value atop the test), config is "nested" for each test case by SapphireTest - you can feel free to modify the config as you see fit, nothing will "last" (bleed out, affect other tests, etc.) beyond the end of the test case :)

Reply:
removed